### PR TITLE
Assignment 1 pull request for Lingfeng Huang

### PR DESCRIPTION
**Pull request by lhuang58**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,20 +66,21 @@ People
 * [Zeeshan Hanif](people/zeeshan-hanif.md) - zh278 - [https://s3.us-east-2.amazonaws.com/ctcs53561/grading.html](https://s3.us-east-2.amazonaws.com/ctcs53561/grading.html)
 * [I Huang](people/i-huang.md) - ih265 - [http://ih265.s3-website-us-east-1.amazonaws.com/grading.html](http://ih265.s3-website-us-east-1.amazonaws.com/grading.html)
 * [Matthew Hsu](people/matthew-hsu.md) - mdh267 - [http://cs5356-mdh267.s3-website-us-east-1.amazonaws.com/grading.html](http://cs5356-mdh267.s3-website-us-east-1.amazonaws.com/grading.html)
+* [Lingfeng Huang](people/lingfeng-huang.md) - lh635 - [http://hlingfeng.s3-website.us-east-2.amazonaws.com/grading.html](http://hlingfeng.s3-website.us-east-2.amazonaws.com/grading.html)
```
Comment (amfleming) [inline README.md:75]: Please only change your own lines

Probably you are using a tool that removes trailing whitespace when you save a file

**Review by amfleming — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (lhuang58) [reply README.md:75]: I think now it is been fixed and also I resolved the conflicts.

**Comment by amfleming** [comment]
Shoot looks like some conflicts - can you resolve?

**Comment by lhuang58** [comment]
Conflict is resolved.

**Pull request merged**